### PR TITLE
Cleaning up quick code

### PR DESCRIPTION
**Pull request by emma-nechamkin**
Taking a minute to clean up / resubmit for review some of the quick and dirty changes from launch.

### CODE REVIEW

File: data/data-pipeline/data_pipeline/etl/score/etl_score.py [hunk 1]
```diff
@@ -298,33 +298,19 @@ def _add_percentiles_to_df(
...
                 f"{output_column_name_root}"
                 f"{field_names.PERCENTILE_FIELD_SUFFIX}"
-            ] = np.where(
-                df[field_names.AGRICULTURAL_VALUE_BOOL_FIELD].isna(),
-                np.nan,
-                df[field_names.GEOID_TRACT_FIELD]
-                .map(temporary_ranking)
-                .fillna(0),
+            ] = (
+                df.where(df[field_names.AGRICULTURAL_VALUE_BOOL_FIELD] == True)[
```
Comment (emma-nechamkin) [inline data/data-pipeline/data_pipeline/etl/score/etl_score.py:307]: this should be much cleaner. asserted that it was equal to the existing code.

File: data/data-pipeline/data_pipeline/score/score_m.py [hunk 2]
```diff
@@ -195,7 +196,8 @@ def _climate_factor(self) -> bool:
         )
 
         self._increment_total_eligibility_exceeded(
-            climate_eligibility_columns, skip_fips=("72")
+            climate_eligibility_columns,
```
Comment (emma-nechamkin) [inline data/data-pipeline/data_pipeline/score/score_m.py:199]: this should get rid of the bad code smell!

File: data/data-pipeline/data_pipeline/etl/score/etl_score.py (hunk 1 above)
Comment (emma-nechamkin) [reply data/data-pipeline/data_pipeline/etl/score/etl_score.py:307]: `== True` is gross but the bool has `NaN`s that we need to keep, so it's necessary unfortunately.

File: data/data-pipeline/pyproject.toml [hunk 3]
```diff
@@ -39,6 +39,7 @@ tox = "^3.24.0"
 pytest-mock = "^3.6.1"
 tox-poetry = "^0.4.1"
 pandas-vet = "^0.2.2"
+nb-black = "^1.0.7"
```
Comment (emma-nechamkin) [inline data/data-pipeline/pyproject.toml:42]: added this from the command line (`$ poetry add -D nb_black`)

**Review by esfoobar-usds — CHANGES_REQUESTED** [review]
PR is not passing.

You can patch this commit or make these one line change: https://github.com/usds/justice40-tool/pull/1252/commits/64d614bca00eb3989dc764e75abedf7662dc1e11

**Review by esfoobar-usds — APPROVED** [review]
🍏

**Pull request merged**